### ISSUE TITLE
App icons displayed in different icon packs in the home screen and in app drawer if "Themed Icons" option is enabled.

### ISSUE THREAD

**Issue by vaisakhv** [Deletescape-Media/Lawnchair#2404]
At some point of time my lawnchair installation started showing icons from "[Pix You Android 12 Dark Icons]( https://play.google.com/store/apps/details?id=com.arjunarora.pix_you_cornflower_dark_icons )" icon pack in the home screen instead of the one in the settings.

Since I used the icon pack previously, I force stopped lawnchair and cleared the cache, but it didn't have any effect. Following that I cleared the cache and storage which reverted everything (including home screen icons) back to default. 

But now if I enable "Themed Icons" it is being changed to Pix You icon pack.
After this I cleared the storage and cache of the mentioned icon pack and uninstalled it.
But the issue remained the same.

It seems like this issue mostly occurs only with certain Google apps (not sure).

Screenshots below 👇

- Home screen icons before enabling "Themed Icons" option.

![Screenshot_20220103_013607-01.jpeg](https://user-images.githubusercontent.com/20832588/147888700-311a4655-5305-4a1d-b5fe-a7746f26958a.jpeg)

- Home screen after enabling Themed Icons.

![Screenshot_20220103_013555-01.jpeg](https://user-images.githubusercontent.com/20832588/147888780-798c395b-6224-43f2-8c84-3e36a629cf37.jpeg)

- Changes made until issue was reproduced
![Screenshot_2022_0103_023206-01.jpeg](https://user-images.githubusercontent.com/20832588/147889409-4cb9ac24-c6ce-45c9-8086-d521b28a02d1.jpeg)

The only changes made until the issue occurred was enabling the notification dots(upon later testing: this has no impact on the issue) access and enabling Themed Icons option.

- App drawer after enabling Themed Icons.

![Screenshot_20220103_015642-01.jpeg](https://user-images.githubusercontent.com/20832588/147888808-18db446a-2c89-4816-b5ee-c19d47bc6013.jpeg)

My environment
- Device model : Iqoo 7
- Manufacturer : Vivo
- Android Skin : Funtouch OS 12
- Android version : 12
- Lawnchair : v12 alpha 04

**Comment by suphon-t** [comment]
What's the problem here? This is exactly what "themed icons" is supposed to do.

**Comment by vaisakhv** [comment]
> What's the problem here? This is exactly what "themed icons" is supposed to do. 

If this is "the feature" then it's still an issue. Because none of my other icon are not being "themed". Please find the below screenshot with some common apps. As I said earlier only some (not all) Google owned app icons are getting a different theme.

![Screenshot_20220103_222008-01.jpeg](https://user-images.githubusercontent.com/20832588/147958247-f722b315-f53d-415c-98c0-502b1f444ad2.jpeg)

**Comment by RikKoedoot** [comment]
Not all apps are supported with themed icons, this is normal and also not a bug

**Comment by vaisakhv** [comment]
> Not all apps are supported with themed icons, this is normal and also not a bug

I've never enabled the themed icons option. I was unaware of the feature.

For me, there is only a handful of Google apps that are getting themed and I've almost   100 apps installed. I tried adding most of them to the home screen just to check.

So when this happened what i thought was, if only 10 to 15 app icons that too from the same dev is getting changed that might be an issue. 

Tldr, I understand that this is not a bug. But if it's only affecting .5% this feature is not a "feature" as such. It's more of a bug 

No disrespect here.... ✌️

**Comment by RikKoedoot** [comment]
We implemented the apps Google added themed icons for, so it works like intended. We are working on adding more apps via an addon (not released yet)
